index.js

```diff
@@ -0,0 +1,199 @@
+let sticky = tabs.offsetTop;
+let stickyMain = headerInner.offsetTop;
+let currentSlider = 0;
+let previousPageYOffset = 0;
+let linkPageYOffset = {
+  headTips: document.getElementById("headTips").offsetTop,
+  headGlossary: document.getElementById("headGlossary").offsetTop,
+  headFAQs: document.getElementById("headFAQs").offsetTop,
+};
+createAccordionTop(accordContainer);
+createAccordionBottom(headAccordionFaqs);
+createSliderClick(newArrSliderText);
+
+function createSliderClick(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    let div = document.createElement("div");
+    let span = document.createElement("span");
+    span.innerText = arr[i].name;
+    div.classList.add("sliderItem");
+    div.appendChild(span);
+    sliderHead.appendChild(div);
+    div.addEventListener("click", changeSliderText);
+    if (i === currentSlider) {
+      div.classList.add("activeSliderLink");
+    }
+  }
+}
+
+function changeSliderText() {
+  Array.from(sliderHead.children).forEach((slider) => {
+    slider.classList.remove("activeSliderLink");
+  });
+  newArrSliderText.forEach((slider, i) => {
+    if (this.children[0].innerText === slider.name) {
+      sliderHead.children[i].classList.add("activeSliderLink");
+      currentSlider = i;
+      console.log(currentSlider);
+      sliderP.innerText = slider.name;
+      sliderSpan.innerText = slider.text;
+    }
+  });
+}
+
+function changeSliderTextArrow() {
+  let nextNumber = this.value === "right" ? 1 : -1;
+  currentSlider += nextNumber;
+  if (currentSlider + nextNumber > newArrSliderText.length) {
+    currentSlider = 0;
+  }
+  if (currentSlider === -1) {
+    currentSlider = newArrSliderText.length - 1;
+  }
+  sliderP.innerText = newArrSliderText[currentSlider].name;
+  sliderSpan.innerText = newArrSliderText[currentSlider].text;
+  Array.from(sliderHead.children).forEach((slider) => {
+    slider.classList.remove("activeSliderLink");
+  });
+  sliderHead.children[currentSlider].classList.add("activeSliderLink");
+}
+
+for (let i = 0; i < sliderButton.length; i++) {
+  sliderButton[i].addEventListener("click", changeSliderTextArrow);
+}
+
+gamburgerContainer.addEventListener("click", () => {
+  containerHeaderMenu.classList.toggle("none");
+  gamburger.classList.toggle("gamburgerIcon");
+  gamburger.classList.toggle("backIcon");
+  document.body.classList.toggle("overflow");
+});
+
+function createAccordionTop(acc) {
+  for (let i = 0; i < acc.length; i++) {
+    acc[i].addEventListener("click", (e) => {
+      let panel = e.currentTarget.children[1];
+      e.currentTarget.children[0].children[2].classList.toggle(
+        "accordionActive"
+      );
+      e.currentTarget.children[0].children[2].classList.toggle("accordion");
+      e.currentTarget.children[0].children[0].classList.toggle("none");
+      e.currentTarget.children[0].children[1].classList.toggle(
+        "accordTextActive"
+      );
+      if (panel.style.maxHeight) {
```

Comment:
wgy do you need a panel height?

Reply:
because if panel height = 0, i adding maxHeight, to open the accordion, and removing if height != 0

Reply:
what if the panel content will be more than 800 px? create just css class for ex `is-open` and when dropdown was clicked add this class and show the panel on the second click remove this class